bot.go

```diff
@@ -630,3 +645,35 @@ func messageReactionAdd(s *discordgo.Session, r *discordgo.MessageReactionAdd) {
 		}
 	}
 }
+
+func IsAdmin(userID string) bool {
+	if BotAdmin == "" {
+		return false
+	}
+	return BotAdmin == userID
+}
+
+func scheduleReset(s *discordgo.Session) {
+	go func() {
+		registeredCommands, err := s.ApplicationCommands(s.State.User.ID, "")
+		if err != nil {
+			log.Critf("Could not fetch registered commands: %v", err)
+		}
+		for _, v := range registeredCommands {
+			err = s.ApplicationCommandDelete(s.State.User.ID, "", v.ID)
+			if err != nil {
+				log.Critf("Cannot delete '%v' command: %v", v.Name, err)
+			}
+		}
+		log.Infof("All %d commands deleted, waiting 3 seconds before resetting", len(registeredCommands))
+		time.Sleep(3 * time.Second)
```

Comment:
I would use this


```suggestion
		delay := 3 * time.Second
		log.Infof("All %d commands deleted, waiting %s before resetting", len(registeredCommands), delay.String())
		time.Sleep(delay)
```

Because having two lines that need to be kept in sync is a bad idea

Reply:
Changing, thanks, note that duration implements stringer so the .String() is not needed

Reply:
I knew it, but I was lazy